android/app/src/main/java/com/converse/dev/BackendApi.kt

```diff
@@ -55,13 +57,13 @@ private suspend fun makePutRequest(appContext: Context, apiURI: String, address:
 
 suspend fun putGroupInviteRequest(appContext: Context, apiURI: String, xmtpClient: Client, joinRequestId: String, status: String) {
     val groupInviteEndpoint = "$apiURI/api/groupJoinRequest/$joinRequestId"
-    val privateKey = PrivateKeyBuilder.buildFromSignedPrivateKey(xmtpClient.keys.identityKey)
-    val signature = Base64.encodeToString(PrivateKeyBuilder(privateKey).sign("XMTP_IDENTITY".toByteArray()).toByteArray(), NO_WRAP)
+
+    val messageToSign = "XMTP_IDENTITY".toByteArray(Charsets.UTF_8)
+    val signature = Base64.encodeToString(xmtpClient.keys.identityKey.sign(Util.keccak256(messageToSign)).toByteArray(), NO_WRAP)
```

Comment:
It looks like[ in JS we create a signature proto](https://github.com/ephemeraHQ/converse-app/blob/ededda0108bcd63d7803f942c0cac7df757b666e/utils/xmtpRN/api.ts#L34) and then base64 encode that, not just base64 encoding the raw signature bytes

Reply:
I think this is base61 encoding the signature proto since thats whats returned by the sign function.

Reply:
Oh, weird. OK. Then that's fine